flows/load_flows_concurrently.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+
+from prefect import Flow
+from prefect.runner.storage import GitRepository
+
+
+async def load_flow(entrypoint: str) -> Flow[..., object]:
```

Comment:
why `object`? This means the caller can't do anything with the return type of the flow (except perhaps print it).

- if you can't determine a type with supported type syntax, use `Any`.
- if the code that wants to use the flow return value is not ever going to use it or if operations that are supported by `object` are the only things they are permitted to do, use `object`.

Note that it doesn't matter what the caller actually does with the returned flow or the flow result; that's up to the caller.

I think this should really be `Any`.